### PR TITLE
feat: MET-2168 add methods to handle highlighting

### PR DESCRIPTION
**Pull request by dawidsdh**
This PR will implement methods to getObjectByPixel and highlightMeshes.

### CODE REVIEW

File: src/facade/index.ts [hunk 1]
```diff
@@ -290,33 +289,79 @@ export const cherryFacade = (cherryViewer: CherryViewer) => {
...
+      ids.forEach((id) => {
+        hoverObject.setParameter(id, parameter, val);
+        pm.isDirty = true;
+
+        // Used to remove highlight after 2 sec.
+        if (hasTimeout) {
+          setTimeout(() => {
+            hoverObject.setParameter(id, parameter, false);
+            pm.isDirty = true;
+          }, 2000);
```
Comment (willowish) [inline src/facade/index.ts:357]: why 2000 ms? At least extract it into a variable

File: src/facade/index.ts [hunk 2]
```diff
@@ -290,33 +289,79 @@ export const cherryFacade = (cherryViewer: CherryViewer) => {
...
-    meshType: 'MESH' | 'MATERIAL' | 'GROUP'
-  ) => {
+    hasTimeout: boolean = false
+  ): CherryKey | null => {
     const parameter = 'highlighted';
-    const object = scene.getObject(key);
-    object.setParameter(index, parameter, value);
-    pm.isDirty = true;
+    // If timeout is not set the value is taken form prop `value` else it is by default set to true
+    const val = hasTimeout ? true : value;
```
Comment (willowish) [inline src/facade/index.ts:343]: maybe name it in a more meaningful way? `val` is pretty ambiguous.

File: src/facade/index.ts [hunk 3]
```diff
@@ -401,20 +440,83 @@ export const cherryFacade = (cherryViewer: CherryViewer) => {
...
+    const gizmo = scene.addObject(GIZMO_ROTATE_KEY, 'assets/rotate.c3b');
+    gizmo.setParameter('visible', false);
+    gizmo.setParameter('gizmo', true);
+
+    const meshes: Record<string, number> = {};
+    const objectMeshes = gizmo.getMeshes();
+    for (let index = 0; index < objectMeshes.size(); index++) {
+      const { mesh_id, mesh_name } = objectMeshes.get(index);
+      meshes[mesh_name] = mesh_id;
+    }
```
Comment (willowish) [inline src/facade/index.ts:489]: this code (445-453) is repeated twice. Extract it into a method.

**Review by willowish — CHANGES_REQUESTED** [review]

File: src/facade/index.ts (hunk 1 above)
Comment (dawidsdh) [reply src/facade/index.ts:357]: The same time we have in the old editor so I added it also here.

**Review by damian08kk — APPROVED** [review]

**Pull request merged**